kubernetes/GKE/README.md

```diff
@@ -6,16 +6,31 @@ This quickstart requires that you are running the gcloud version 319.0.0 or late
 
 #### 1.1 Set Gcloud account information
 
+Create a gcloud configuration with account and project info.
+
 ```bash
 gcloud init
 ```
 
+Set compute/region and compute/zone in config
+
+```bash
+gcloud config set compute/region us-west1
+gcloud config set compute/zone us-west1-a
+```
+
+If you have multiple configurations, activate required config.
+
+```bash
+gcloud config configurations activate <config_name>
+```
+
 #### 1.2 Create GKE cluster
 
 Use the [gcloud container clusters create](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-zonal-cluster) command to create an GKE cluster. The following example creates a cluster named *torchserve* with one node with a *nvidia-tesla-t4* GPU. This will take several minutes to complete.
 
 ```bash
-gcloud container clusters create torchserve --machine-type n1-standard-4 --accelerator type=nvidia-tesla-t4,count=1 --num-nodes 1 --region us-west1 --node-locations us-west1-a
+gcloud container clusters create torchserve --machine-type n1-standard-4 --accelerator type=nvidia-tesla-t4,count=1 --num-nodes 1
```

Comment:
Will these steps work if no explicit region is specified?

Also add a note that "accelerator" parameter is optional, use it for creating cluster for GPU nodes. It will be help to include example of command for CPU cluster to remove an confusion

Reply:
Yes, the region will be set from glcoud config.